bin/oneoff/remove_password_for_code_studio_admin.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# Remove passwords for code studio admins
+
+batch_number = 0
+User.where(admin: true).where.not(encrypted_password: nil).find_in_batches(batch_size: 10) do |batch|
```

Comment:
will this query complete in time? Given that you had to use SQL to execute the `count` query, I would assume the same problem will present itself when you try to run this script.

It also seems unnecessary to batch a query that we expect to run on fewer than 50 rows.

Reply:
I thought about this and went the route of using a transaction because it was considered a best practice.  However, I do agree that it is not necessary in this case and that catching and logging the error will be more effective.

Reply:
note that transactions are used for a different concern; specifically, they ensure that you can group queries together such that if any one query fails, all others will be rolled back

Reply:
@Hamms You mention concern with how long the query will take.  Any ideas on how to tackle this?

Reply:
If the query is too expensive to fit in the 30 second timeout, it's recommended to write the script in such a way that it can accept a file containing a list of ids to update; you can then run the query against the reporting database to generate that list.

Reply:
@Hamms like any of the examples below?
  
https://github.com/code-dot-org/code-dot-org/pull/31828/files
Or
https://github.com/code-dot-org/code-dot-org/blob/staging/bin/oneoff/wipe_data/remove_unused_production_users